alchemy/src/cloudflare/miniflare/remote-binding-proxy.ts

```diff
@@ -162,7 +180,12 @@ async function createWorkersPreviewSession(api: CloudflareApi) {
     "create workers preview session",
     api.get(`/accounts/${api.accountId}/workers/subdomain/edge-preview`),
   );
-  const res = await fetch(exchange_url);
+  const accessToken = await getAccessToken(new URL(exchange_url).hostname);
```

Comment:
Why is `getAccessToken` called three times?

1. `exchange_url` - some randomly generated `workers.dev` URL (e.g. `<random-string>.chipotle.workers.dev`), otherwise you get an access error on the domain
2. `prewarm` - seems like the same domain as the `exchange_url`, but can't tell if that's guaranteed; Wrangler just calls `getAccessToken` separately but caches the result, which we do too by wrapping the function in `memoize`
3. The actual worker URL (e.g. `my-worker.chipotle.workers.dev`)

Reply:
Are you asking or explaining?